statistics-dashboard-rest/src/main/java/statistics/dashboard/rest/config/StatisticsRestInitializer.java

```diff
@@ -0,0 +1,44 @@
+package statistics.dashboard.rest.config;
+
+import java.io.File;
+import javax.servlet.MultipartConfigElement;
+import javax.servlet.ServletRegistration.Dynamic;
+import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;
+
+/**
+ * This class is the bootstrap code for Spring. It tells Spring how to start this web application.
+ */
+public class StatisticsRestInitializer extends
+        AbstractAnnotationConfigDispatcherServletInitializer {
+
+  private static final int MAX_UPLOAD_SIZE_IN_MB = 5 * 1024 * 1024; // 5 MB
+
+  @Override
+  protected Class<?>[] getRootConfigClasses() {
+    return new Class<?>[0];
+  }
+
+  @Override
+  protected Class<?>[] getServletConfigClasses() {
+    return new Class[]{StatisticsRestApplication.class};
+  }
+
+  @Override
+  protected String[] getServletMappings() {
+    return new String[]{"/"};
+  }
+
+  @Override
+  protected void customizeRegistration(Dynamic registration) {
```

Comment:
I don't think it needs this bit? We're not doing file uploads or multipart form data, I think? Nor are we likely to do so soon.

Reply:
Thanks for pointing it out, I'll remove this